tests/perf/test_classification.py

```diff
@@ -258,3 +261,121 @@ def test_perf(
             criteria=self.BENCHMARK_CRITERIA,
             resume_from=fxt_resume_from,
         )
+
+
+class TestPerfSemiSLMultiClass(PerfTestBase):
+    """Benchmark single-label classification for Semi-SL task."""
+
+    MODEL_TEST_CASES = [  # noqa: RUF012
+        Benchmark.Model(task="classification/multi_class_cls", name="efficientnet_b0_semisl", category="speed"),
```

Comment:
It seems that two models are set to `speed`. Is this your intention?

Reply:
I'll update this later when see the results.